CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
```

Comment:
Looks like this change https://github.com/signalfx/splunk-otel-collector-chart/pull/136 is missed. Can you add it please?